vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -86,33 +87,32 @@ string HResultToString(hresult &&result) {
 
 namespace Microsoft::React {
 
+// private
+WinRTWebSocketResource::WinRTWebSocketResource(
+    IMessageWebSocket &&socket,
+    Uri &&uri,
+    vector<ChainValidationResult> &&certExceptions)
+    : WinRTWebSocketResource(
+          std::move(socket),
+          DataWriter{socket.OutputStream()},
+          std::move(uri),
+          std::move(certExceptions)) {}
+
 WinRTWebSocketResource::WinRTWebSocketResource(
     IMessageWebSocket &&socket,
     IDataWriter &&writer,
     Uri &&uri,
-    vector<ChainValidationResult> &&certExeptions) noexcept
```

Comment:
>noexcept [](http://example.com/codeflow?start=51&length=8)

(nit) why? which exceptions are expected to be thrown from here?

Reply:
2 reported at least:
1. Malformed `Uri` (i.e. `Uri{""}`)
2. Plain websocket instantiation failure (unknown cause): `MessageWebSocket{}`

Reply:
Maybe worth adding a comment that such exception types could be thrown, so that consumers are at least aware?

Reply:
I wish the WinRT docs did that in the first place 😅

This should not be necessary because after this fix the exception will propagate to the JS layer and can be handled gracefully.

While `2.`'s causes are still unclear, `1.` is actually very expected if a malformed URI is passed to a component meant to establish a connection.